trio/_subprocess.py

```diff
@@ -482,24 +535,26 @@ async def run_process(
           be a string, which will be parsed following platform-dependent
           :ref:`quoting rules <subprocess-quoting>`.
 
-      stdin (:obj:`bytes`, file descriptor, or None): The bytes to provide to
-          the subprocess on its standard input stream, or ``None`` if the
-          subprocess's standard input should come from the same place as
-          the parent Trio process's standard input. As is the case with
-          the :mod:`subprocess` module, you can also pass a
-          file descriptor or an object with a ``fileno()`` method,
-          in which case the subprocess's standard input will come from
-          that file.
+      stdin (:obj:`bytes`, subprocess.PIPE, file descriptor, or None): The
+          bytes to provide to the subprocess on its standard input stream, or
+          ``None`` if the subprocess's standard input should come from the
+          same place as the parent Trio process's standard input. As is the
+          case with the :mod:`subprocess` module, you can also pass a file
+          descriptor or an object with a ``fileno()`` method, in which case
+          the subprocess's standard input will come from that file. And when
+          starting `run_process` as a background task, you can use
+          ``stdin=subprocess.PIPE``, in which case `Process.stdin` will be a
+          `~trio.abc.SendStream` that you can use to send data to the child.
```

Comment:
For most of these docs, there is a visual distinction (indentation, newlines) between info related to `await run_process` vs `await nursery.start(run_process)` but it is kind of lacking here. On the other hand, I don't know if sphinx will permit a newline here?